### PR TITLE
Update BDR to 3.7.12; add pglogical

### PR DESCRIPTION
**Pull request by josh-heyer**
## What Changed?

This updates the BDR docs to version 3.7.12 and adds pglogical as a product (also at version 3.7.12)

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [x] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content
- [x] This PR is for a release, please add this tag: `product/bdr/3.7.12`

### CODE REVIEW

**Comment by josh-heyer** [comment]
Ok, preview build now includes pglogical, @drothery-edb

File: product_docs/docs/pglogical/3.7/configuration.mdx [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+the replicated changes, enough worker process slots need to be present:
+
+```
+max_worker_processes = 10
+```
+
+The formula for computing the correct value of `max_worker_processes` is:
+one for instance + one per database on the provider (upstream), one for instance
+
+-   one per database + two per subscription on the subscriber (downstream).
```
Comment (drothery-edb) [inline product_docs/docs/pglogical/3.7/configuration.mdx:34]: this rendered incorrectly. it should be one paragraph without a bullet in the middle: 

The formula for computing the correct value of max_worker_processes is: one for instance + one per database on the provider (upstream), one for instance + one per database + two per subscription on the subscriber (downstream).

**Review by drothery-edb — APPROVED** [review]
I found one small thing in the pglogical content. Not sure it is worth fixing. Took a quick look at the BDR content and that seemed fine.

File: product_docs/docs/pglogical/3.7/configuration.mdx (hunk 1 above)
Comment (josh-heyer) [reply product_docs/docs/pglogical/3.7/configuration.mdx]: Fixing this in situ for now; smells like a parser bug, but I'll have to investigate another time.

**Pull request merged**